### PR TITLE
Observation vectors in config yaml

### LINKED ISSUE #145: Observation errors in flownet config YAML
The user now needs to define the observation error in the FlowNet config YAML. Currently, relative (and minimum errors) for all of the vectors WOPR, WGPR, WWPR, WBHP, WTHP needs to be specified. This should be changed to only specifying for the vectors the user wants to condition on.

### PR DESCRIPTION
**Pull request by olelod**
Testing potential implications of removing WGPR from Egg ci_config YAML on testdata repository

- [x] :tada: This PR closes #145 and #153 .
- [x] :scroll: I have broken down my PR into the following tasks:
   - [x] Make sure that only the vectors used for HM needs to be defined in FlowNet config YAML 
   - [x] Add more vectors (WWIR and WGIR) to _config_parser.py (and observations.yamlobs/ertobs)
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [x] :book: I have considered adding a new entry in `CHANGELOG.md`.
- [ ] :books: I have considered updating the documentation.

### CODE REVIEW

**Comment by olelod** [comment]
Closes #145 and #153

File: src/flownet/templates/observations.ertobs.jinja2 [hunk 1]
```diff
@@ -34,13 +37,18 @@ SUMMARY_OBSERVATION WTHP_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.thp
...
 {%- endif %}
-{%- if not isnan(kw.thp): %}
-SUMMARY_OBSERVATION WTHP_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.thp }}; ERROR = {{ [0.05 * kw.thp, 5] | max }}; DATE = {{ date_formatted  }}; KEY = WTHP:{{ kw.well_name }}; };
+{%- if not isnan(kw.rate) and kw.inj_type == "GAS" and error_config.WGIR.rel_error is not none and error_config.WGIR.min_error is not none: %}
+SUMMARY_OBSERVATION WGIR_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.rate }}; ERROR = {{ [error_config.WGIR.rel_error * kw.rate, error_config.WGIR.min_error] | max }}; DATE = {{ date_formatted  }}; KEY = WGIR:{{ kw.well_name }}; };
+{%- endif %}
+{%- if not isnan(kw.rate) and kw.inj_type == "WATER" and error_config.WWIR.rel_error is not none and error_config.WWIR.min_error is not none: %}
+SUMMARY_OBSERVATION WWIR_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.rate }}; ERROR = {{ [error_config.WWIR.rel_error * kw.rate, error_config.WWIR.min_error] | max }}; DATE = {{ date_formatted  }}; KEY = WGIR:{{ kw.well_name }}; };
```
Comment (wouterjdb) [inline src/flownet/templates/observations.ertobs.jinja2:50]:
```suggestion
SUMMARY_OBSERVATION WWIR_{{ kw.well_name }}_{{ index_name }} { VALUE = {{ kw.rate }}; ERROR = {{ [error_config.WWIR.rel_error * kw.rate, error_config.WWIR.min_error] | max }}; DATE = {{ date_formatted  }}; KEY = WWIR:{{ kw.well_name }}; };
```

**Review by wouterjdb — CHANGES_REQUESTED** [review]

File: src/flownet/templates/observations.yamlobs.jinja2 [hunk 2]
```diff
@@ -50,27 +59,56 @@ smry:
...
+   {%- if (error_config.WTHP.rel_error is not none) and (error_config.WTHP.min_error is not none) -%}
    {%- for kw in schedule.get_keywords(kw_class="WCONINJH", well_name=well_name, ignore_nan="thp") -%}
    {% if loop.first and not loop.last: %}
  - key: WTHP:{{ well_name }}
    observations:
    {%- elif not loop.first: %}
     - date: {{ kw.date.strftime('%Y-%m-%d') }}
       value: {{ kw.bhp }}
-      error: {{ [0.05 * kw.thp, 5] | max }}
+      error: {{ [error_config.WTHP.rel_error * kw.bhp, error_config.WTHP.min_error] | max }}
```
Comment (wouterjdb) [inline src/flownet/templates/observations.yamlobs.jinja2:86]: This should be THP I guess?

```suggestion
      error: {{ [error_config.WTHP.rel_error * kw.thp, error_config.WTHP.min_error] | max }}
```

**Review by wouterjdb — CHANGES_REQUESTED** [review]

**Review by wouterjdb — APPROVED** [review]
LGTM 🚀

**Pull request merged**